src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy

```diff
@@ -195,30 +195,44 @@ class BugsnagPlugin implements Plugin<Project> {
     }
 
     /**
-     * Fetches all the assemble tasks in the current project that match the variant
+     * Fetches all the assemble and bundle tasks in the current project that match the variant
      *
-     * Expected behaviour: assemble, assembleJavaExampleRelease, assembleJavaExample, assembleRelease
+     * Expected behaviour: [assemble, assembleJavaExampleRelease, assembleJavaExample, assembleRelease,
+     * bundle, bundleJavaExampleRelease, bundleJavaExample, bundleRelease]
      *
      * @param output the variantOutput
      * @param project the current project
-     * @return the assemble tasks
+     * @return the assemble + bundle tasks
      */
-    private static Set<Task> findAssembleTasks(BaseVariantOutput output, Project project) {
+    private static Set<Task> findAssembleBundleTasks(BaseVariantOutput output, Project project) {
+        Set<String> taskNames = new HashSet<>()
+        taskNames.addAll(findTaskNamesForPrefix(output, "assemble"))
+        taskNames.addAll(findTaskNamesForPrefix(output, "bundle"))
+
+        project.tasks.findAll {
+            taskNames.contains(it.name)
+        }
+    }
+
+    /**
+     * Finds all the task names which can be used to assemble a variant, and replaces 'assemble' with the given
+     * prefix.
+     *
+     * E.g. [bundle, bundleRelease, bundleFooRelease]
+     */
+    private static Set<String> findTaskNamesForPrefix(BaseVariantOutput output, String prefix) {
         String variantName = output.name.split("-")[0].capitalize()
         String assembleTaskName = output.assemble.name
```

Comment:
It's a shame we can't get the "bundle" task from the build variant instead of using the assemble one and replacing the names.